README.md

```diff
@@ -1,16 +1,33 @@
-# Lane Keeping Assist via CNN
+# Lane Keeping Assist
+Welcome!
 
-### Repository Structure
+## :classical_building: Repository Structure
 ```bash
 ├── README.md       
 ├── subject.pdf   # subject of the project
 │ 
 ├── src/          # .py .ipynb files for building a model and control a car
 │   ├── main.py
-│
 │── docs/         # .md files documentations for the project
 │   ├── CNN.md 
 │   ├── Carla.md
 │   ├── Control_car.md
 │   ├── Data_processing.md
 ```
+
+## :card_file_box: Table of Contents
+- [Lane Keeping Assist](#lane-keeping-assist)
+  - [:classical\_building: Repository Structure](#classical_building-repository-structure)
+  - [:card\_file\_box: Table of Contents](#card_file_box-table-of-contents)
+  - [:computer: Tech Stacks](#computer-tech-stacks)
+  - [:goal\_net: Project Goal](#goal_net-project-goal)
+  - [:office\_worker: Contributor](#office_worker-contributor)
```

Comment:
These icons are an eyesore, please delete them

Reply:
Including meaningful emoji looks better and also understandable. Now there is only titles so it could be annoying, but when we add contents to them, it will be better.

Reply:
But it's autogenerated, not by mine

Reply:
They auto-update following real title. So if you want, we can remove this at final version. Deleting this everytime is annoying

Reply:
you can just delete them by yourself

> But it's autogenerated, not by mine

Reply:
🤢 solved